### PR TITLE
[TASK] #152 Fix TCA sort by

### PR DESCRIPTION
**Pull request by kpurrmann**

### CODE REVIEW

File: Configuration/TCA/tx_typo3forum_domain_model_user_notification.php [hunk 1]
```diff
@@ -11,7 +11,7 @@
 		'type' => 'type',
 		'crdate' => 'crdate',
 		'delete' => 'deleted',
-		'sortby' => 'crdate DESC',
+		'sortby' => 'ORDER BY crdate',
```
Comment (martin-helmich) [inline Configuration/TCA/tx_typo3forum_domain_model_user_notification.php]: Are you sure that `sortby` requires the entire `ORDER BY` statement? [The docs](https://docs.typo3.org/typo3cms/TCAReference/Reference/Ctrl/Index.html) are not especially clear on this, but it seems that just the field name is required. Or should this be `default_sortby`, also?

File: Configuration/TCA/tx_typo3forum_domain_model_stats_summary.php [hunk 2]
```diff
@@ -10,7 +10,7 @@
 		'label_alt_force' => true,
 		'tstamp' => 'tstamp',
 		'delete' => 'deleted',
-		'sortby' => 'tstamp DESC',
+		'default_sortby' => 'ORDER BY tstamp',
```
Comment (martin-helmich) [inline Configuration/TCA/tx_typo3forum_domain_model_stats_summary.php]: Is the reversion of sort order (was `DESC`, now implicitly `ASC`) intentional?

File: Configuration/TCA/tx_typo3forum_domain_model_user_notification.php (hunk 1 above)
Comment (martin-helmich) [inline Configuration/TCA/tx_typo3forum_domain_model_user_notification.php]: Is the reversion of sort order (was `DESC`, now implicitly `ASC`) intentional?

File: Configuration/TCA/tx_typo3forum_domain_model_stats_summary.php (hunk 2 above)
Comment (kpurrmann) [inline Configuration/TCA/tx_typo3forum_domain_model_stats_summary.php]: Checked it. Added missing `DESC`

**Pull request merged**